x-pack/plugins/synthetics/e2e/synthetics/journeys/private_locations.journey.ts

```diff
@@ -159,8 +159,9 @@ journey(`PrivateLocationsSettings`, async ({ page, params }) => {
   step('viewer user cannot add locations', async () => {
     await syntheticsApp.navigateToSettings(false);
     await page.click('text=Private Locations');
+    await page.hover(byTestId('syntheticsEmptyLocationsButton'), { force: true });
     await page.waitForSelector(
-      `text="You're missing some Kibana privileges to manage private locations"`
+      `text="You do not have sufficient permissions to perform this action."`
```

Comment:
Is there a reason why we're making the callout title more vague?
It's much clearer if we include at least what "action" or user goal is concerned. 

It could be something like, "There's no agent policy available and you're missing privileges to create one"